### PR TITLE
AF-1799: Upgrade JDK 11 + GWT 2.9

### PR DESCRIPTION
**Pull request by caponetto**
**Upgrade JDK 11 + GWT 2.9**

Feel free to comment and/or raise any concerns.

This PR continues the work done in 
- https://github.com/kiegroup/appformer/pull/1030
- https://github.com/kiegroup/appformer/pull/1039

Thanks guys for that!

**JIRA**:

[AF-1799](https://issues.redhat.com/browse/AF-1799)

**Parent Pull Request**:
- https://github.com/kiegroup/lienzo-core/pull/134

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
 
* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>
  
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

**Comment by caponetto** [comment]
Jenkins retest this

**Comment by caponetto** [comment]
Jenkins run fdb
Jenkins run cdb

**Comment by caponetto** [comment]
Jenkins run cdb

**Comment by caponetto** [comment]
Jenkins run cdb

**Comment by caponetto** [comment]
Jenkins run cdb

**Comment by caponetto** [comment]
Jenkins run cdb

**Comment by caponetto** [comment]
Jenkins run cdb

**Comment by caponetto** [comment]
Jenkins run cdb

File: jbpm-recommendation-pmml-random-forest/pom.xml [hunk 1]
```diff
@@ -50,6 +50,12 @@
           <groupId>org.jbpm</groupId>
           <artifactId>jbpm-test</artifactId>
           <version>${version.org.kie}</version>
```
Comment (Ginxo) [inline jbpm-recommendation-pmml-random-forest/pom.xml:52]: it's not part of the PR itself, but since I notice `${version.org.kie}` it would be nice to remove it
```suggestion
```

**Review by Ginxo — APPROVED** [review]
LGTM, just check my proposal

File: jbpm-recommendation-pmml-random-forest/pom.xml (hunk 1 above)
Comment (caponetto) [reply jbpm-recommendation-pmml-random-forest/pom.xml:52]: Hi @Ginxo, thanks for dedicating some time to review this PR!

There are lots of `<version>${version.org.kie}</version>` in the `pom.xml` files of this repository.
I tried to simply remove them but maven complains about missing versions.
Probably other changes are necessary to achieve this goal.
Can you please create a JIRA ticket and assign it accordingly?

**Review by mareknovotny — APPROVED** [review]

**Review by Srihari1192 — APPROVED** [review]
